routes/web.php

```diff
@@ -41,3 +44,10 @@
 	}
 });
 
+Route::get('/user/edit', function () {
+	return view('editUserProfile');
+});
+
+Route::post('/contact', [
```

Comment:
`Route::post('/contact', 'ContactMessageController@post');` should work (+ simplified)